benches/bench_slice.rs

```diff
@@ -0,0 +1,30 @@
+#![feature(test)]
```

Comment:
Yeah, it's not that useful, it just verifies that internal and external iterators are roughly the same speed. It'll get more interesting once btree iterators are implemented.

Hm, we'll probably have to externalize the btree implementation in order to implement it efficiently, won't we?